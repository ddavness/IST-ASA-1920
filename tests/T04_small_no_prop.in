5,5
16
16
13
12
12
1 2
2 1
2 3
3 5
5 4
