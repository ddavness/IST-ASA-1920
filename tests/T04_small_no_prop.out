16
16
13
12
12
